qt/python/instrumentview/instrumentview/renderers/base_renderer.py

```
@@ -19,11 +19,25 @@ class InstrumentRenderer(ABC):
 
+    # Marker colour for picked detectors.  Deliberately distinct from the monitor
+    # colour (red) and from the counts colour map, so a selection can never be
+    # mistaken for either.
+    _PICKED_HIGHLIGHT_COLOUR = "magenta"
+
+    # Opacity transfer function for the pickable overlay, as [unpicked, picked].
+    # The picked entry is fully transparent: the magenta marker drawn by
+    # update_picked_highlight marks the selection instead, and a tinted fill on
+    # top of it only muddied the colour and hid the detector's counts.  The
+    # overlay actor itself is still needed — it is the pick target.
+    _PICKED_FILL_OPACITY = [0.0, 0.0]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)